.ci/lib.sh

```diff
@@ -123,10 +123,7 @@ function get_dep_from_yaml_db(){
 
 	[ ! -f "$versions_file" ] && die "cannot find $versions_file"
 
-	# directory of this script, not the caller
-	local cidir=$(dirname "${BASH_SOURCE[0]}")
-
-	${cidir}/install_yq.sh >&2
+	"${GOPATH}/src/${tests_repo}/.ci/install_yq.sh" >&2
```

Comment:
I wonder why 'sometimes' it is not found - do you have any specifics @GabyCT ?
@chavafg - can you confirm, this hard wiring of the GOPATH will work in all cases such as our depends-on etc.?

Reply:
@grahamwhaley, here it is an example
```
06:28:41 + .ci/ci_cache_components.sh -c
06:28:41 /tmp/jenkins/workspace/cloud-hypervisor-nightly-x86_64/artifacts /tmp/jenkins/workspace/cloud-hypervisor-nightly-x86_64/go/src/github.com/kata-containers/tests
06:28:41 artifacts:
06:28:41 .ci/lib.sh: line 129: .ci/install_yq.sh: No such file or directory
```

Reply:
I thought that `dirname` was giving the absolute path, but seems that no. 
I think this should be fine in our `Depends-on` scenarios, but we should better test.

Reply:
The function is only used at the tests repository